projects/plugins/boost/changelog/add-seo-plugins-compatibility

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Adds the Critical CSS generation token (jb-generate-critical-css), to the allowed query args lists of Yoast SEO and All in One SEO. This prevents them from removing it and breaking Critical CSS generation.
```

Comment:
As this is going to the changelog, consider something like:
> Critical CSS: Improved compatibility with Yoast SEO and All in One SEO to ensure smooth Critical CSS generation.

Reply:
Thanks! I've updated the changelog.